CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+* @riandyrn
+* @ilhamsyahids
+* @mustafasegf
+* @ProtozoaJr
```

Comment:
This should be in one line like this:
```
* @riandyrn @ilhamsyahids @mustafasegf @ProtozoaJr
```

as stated in [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) unless it will be takes the most precedence (for example when open PR like #34)

Reply:
Ah, thank you so much, @ilhamsyahids!

I thought it was just like a list in Markdown. Apparently, `*` signified the entire code. 🙈

Reply:
Updated in https://github.com/riandyrn/otelchi/pull/33/commits/ed2b9d4c7ba5d74951f7430580e06f3370f16250, @ilhamsyahids. Please recheck it.